packages/clay-management-toolbar/src/DefaultState.tsx

```diff
@@ -0,0 +1,258 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import CreationMenu from './CreationMenu';
+import Item from './Item';
+import ItemList from './ItemList';
+import React, {useState} from 'react';
+import SearchForm from './SearchForm';
+import SearchInput from './SearchInput';
+import {ClayCheckbox} from '@clayui/form';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+import {IPropsBase} from './types';
+
+interface IProps extends IPropsBase {
+	creationMenu?: React.ComponentProps<typeof CreationMenu>['creationMenu'];
+	onCreationMenuClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onSearchValueChange?: (val: string) => void;
+	onSortingButtonClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	searchValue?: string;
+	selectable?: boolean;
+	showCreationMenu?: boolean;
+	sortingURL?: string;
+	viewTypes?: Array<any>;
+}
+
+const DefaultState: React.FunctionComponent<IProps> = ({
+	checkboxChecked = false,
+	creationMenu = false,
+	disabled = false,
+	filterItems,
+	labels = {
+		checkbox: 'Select Items',
+	},
+	onCheckboxChange = () => {},
+	onCreationMenuClick = () => {},
+	onInfoButtonClick = () => {},
+	onSearchValueChange = () => {},
```

Comment:
We can remove the no-op, and use this prop to determine if `showSearch` should be true. This means we can also remove `showSearch`.